lib/pages/pool/add_liquidity/add_liquidity.dart

```diff
@@ -25,77 +28,79 @@ class _AddLiquidityState extends State<AddLiquidity> {
-    double _height = mediaquery.size.height;
-    double _width = mediaquery.size.width;
-    bool isWeb =
+    final mediaquery = MediaQuery.of(context);
+    final _height = mediaquery.size.height;
+    final _width = mediaquery.size.width;
+    final isWeb =
         kIsWeb && (MediaQuery.of(context).orientation == Orientation.landscape);
-    double layoutHgt = _height * 0.8;
-    double layoutWdt = isWeb ? _width * 0.8 : _width * 0.9;
+    final layoutHgt = _height * 0.8;
+    final layoutWdt = isWeb ? _width * 0.8 : _width * 0.9;
 
-    print(_width);
     return BlocBuilder<PoolBloc, PoolState>(
       buildWhen: (previous, current) => current != previous,
       builder: (context, state) {
         final bloc = context.read<PoolBloc>();
         final poolInfo = state.poolPairInfo;
         final balance0 = state.balance0;
         final balance1 = state.balance1;
-        Token? token0 = state.token0;
-        Token? token1 = state.token1;
+        final token0 = state.token0;
+        final token1 = state.token1;
 
         if (state.status == BlocStatus.initial) {
           bloc.add(PageRefreshEvent());
         }
 
         if (widget.token0 != null && widget.token1 != null) {
-          bloc.add(Token0SelectionChanged(token0: widget.token0!));
-          bloc.add(Token1SelectionChanged(token1: widget.token1!));
-          widget.token0 = null;
-          widget.token1 = null;
+          bloc
+            ..add(Token0SelectionChanged(token0: widget.token0!))
+            ..add(Token1SelectionChanged(token1: widget.token1!));
+          widget
+            ..token0 = null
+            ..token1 = null;
         }
 
         TextStyle textStyle(Color color, double size, bool isBold) {
-          if (isBold)
+          if (isBold) {
             return TextStyle(
               color: color,
               fontFamily: 'OpenSans',
               fontSize: size,
               fontWeight: FontWeight.w500,
             );
-          else
+          } else {
             return TextStyle(
               color: color,
               fontFamily: 'OpenSans',
               fontSize: size,
             );
+          }
         }
 
         BoxDecoration boxDecoration(
-            Color col, double rad, double borWid, Color borCol) {
+          Color col,
+          double rad,
+          double borWid,
+          Color borCol,
+        ) {
           return BoxDecoration(
-              color: col,
-              borderRadius: BorderRadius.circular(rad),
-              border: Border.all(color: borCol, width: borWid));
+            color: col,
+            borderRadius: BorderRadius.circular(rad),
+            border: Border.all(color: borCol, width: borWid),
+          );
         }
 
-        isTokenSelected(Token currentToken, int tknNum) {
+        bool isTokenSelected(Token currentToken, int tknNum) {
           if (tknNum == 1) {
             return currentToken.address == token0.address;
           } else {
```

Comment:
Remove comment here

Reply:
Considering the nature of the logic(comparing against predefined set of integers) I feel that comment helps understanding the code, so it doesn't hurt leaving it in.